cypress/support/commands/common.js

```diff
@@ -63,15 +63,42 @@ Cypress.Commands.add('c_login', (options = {}) => {
     app = '',
     backEndProd = false,
     rateLimitCheck = false,
+    enablePasskey = false,
   } = options
   const { loginEmail, loginPassword } = setLoginUser(user, {
     backEndProd: backEndProd,
   })
   if (!(loginEmail && loginPassword)) {
     throw new Error(`User/Password is not on file`)
   }
+
   cy.c_visitResponsive('/endpoint', 'large', { rateLimitCheck: rateLimitCheck })
+  cy.c_setDerivAppEndpoint(app, backEndProd, user).then(() => {
```

Comment:
as per code we do not need to pass backEndProd and user instead need to pass loginEmail, loginPassword
```suggestion
  cy.c_setDerivAppEndpoint(app, loginEmail, loginPassword).then(() => {
```

Reply:
as discussed these will still needed for `c_setDerivAppEndpoint`